examples/windows/kubernetes-ssh.json

```diff
@@ -0,0 +1,54 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes",
+      "orchestratorRelease": "1.13"
+    },
+    "masterProfile": {
+      "count": 1,
+      "dnsPrefix": "",
+      "vmSize": "Standard_D2_v3"
+    },
+    "agentPoolProfiles": [
+      {
+        "name": "windowspool2",
+        "count": 2,
+        "vmSize": "Standard_D2_v3",
+        "availabilityProfile": "AvailabilitySet",
+        "osType": "Windows",
+        "osDiskSizeGB": 128,
+        "extensions": [
+            {
+                "name": "winrm"
+            }
+        ]
+      }
+    ],
+    "windowsProfile": {
+      "adminUsername": "azureuser",
+      "adminPassword": "replacepassword1234$",
+      "sshEnabled": true

```

Comment:
let's add this field description to https://github.com/Azure/aks-engine/blob/master/docs/topics/clusterdefinitions.md instead of adding a new example file for it

Reply:
File removed, edits made to clusterdefinition